lms/templates/certificates/_accomplishment-banner.html

```diff
@@ -30,6 +30,7 @@
                   social_network: 'LinkedIn'
               };
               Logger.log('edx.certificate.shared', data);
+              window.open('${linked_in_url}');
           });
```

Comment:
@talbs @mattdrayer I used the linkedin own JS click listener. let me know is this change ok for you now ? 
Thanks